### PR TITLE
Use podutils for pull-community-verify

### PR DESCRIPTION
**Pull request by nikhita**
Fixes https://github.com/kubernetes/test-infra/issues/13118

Take two of https://github.com/kubernetes/test-infra/pull/13126

/cc @spiffxp @cblecker @cjwagner

### CODE REVIEW

File: config/jobs/kubernetes/community/community-presubmit.yaml [hunk 1]
```diff
@@ -2,21 +2,17 @@ presubmits:
...
-        - "--upload=gs://kubernetes-jenkins/pr-logs"
-        - --scenario=execute
-        - --
-        - make
-        - verify
+      - image: golang:1.11
+        command:
+        - sh
+        - "-c"
+        - "export PATH=$PATH:$GOPATH/bin && make verify"
```
Comment (nikhita) [inline config/jobs/kubernetes/community/community-presubmit.yaml]: Ref: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-pm/enhancements-presubmit.yaml

(earlier error was https://github.com/kubernetes/test-infra/pull/13126#issuecomment-504610676)

Comment (stevekuznetsov) [reply config/jobs/kubernetes/community/community-presubmit.yaml]: @krzyzacy do you remember why we didn't add this to the PATH in general?

File: config/jobs/kubernetes/community/community-presubmit.yaml [hunk 2]
```diff
@@ -2,21 +2,17 @@ presubmits:
   kubernetes/community:
   - name: pull-community-verify
     branches:
-    - master
+    - ^master$
     always_run: true
-    labels:
-      preset-service-account: "true"
+    decorate: true
+    path_alias: k8s.io/community
```
Comment (spiffxp) [inline config/jobs/kubernetes/community/community-presubmit.yaml]: is this needed?

File: config/jobs/kubernetes/community/community-presubmit.yaml (hunk 1 above)
Comment (cblecker) [reply config/jobs/kubernetes/community/community-presubmit.yaml]: If we are swapping the image out here for the upstream go image, then `$GOPATH/bin` is already in the $PATH

**Comment by nikhita** [comment]
Updated, PTAL.

File: config/jobs/kubernetes/community/community-presubmit.yaml [hunk 3]
```diff
@@ -4,18 +4,13 @@ presubmits:
...
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/gcloud-in-go:v20190125-cc5d6ecff3
-        args:
-        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
-        - "--root=/go/src"
-        - "--upload=gs://kubernetes-jenkins/pr-logs"
-        - --scenario=execute
-        - --
+      - image: golang:1.11
```
Comment (spiffxp) [inline config/jobs/kubernetes/community/community-presubmit.yaml]: I was able to get this working with `golang:1.12.5`

File: config/jobs/kubernetes/community/community-presubmit.yaml [hunk 4]
```diff
@@ -4,18 +4,13 @@ presubmits:
...
+        command:
```
Comment (spiffxp) [inline config/jobs/kubernetes/community/community-presubmit.yaml]: Unfortunately I think the previous command was necessary.  For Reasons™, [podutils decoration messes with GOPATH](https://github.com/kubernetes/test-infra/blob/895df89b7e4238125063157842c191dac6f7e58f/prow/pod-utils/decorate/podspec.go#L474), so while `/go/bin` is on PATH, `/home/prow/go/bin` is not. We may be able to get rid of this in our brave new go module world, but at present I think there are some jobs that depend on this

**Comment by nikhita** [comment]
Updated.

**Comment by nikhita** [comment]
The cla context is not green, but the PR is the merge pool :thinking:

**Comment by nikhita** [comment]
Tested here: https://github.com/kubernetes/community/pull/3747#issuecomment-505512599
https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/community/3747/pull-community-verify/1143551215615873034

It worked! :tada:

**Pull request merged**